ros_buildfarm/pulp.py

```diff
@@ -130,7 +130,7 @@ def enumerate_pkgs_in_repo_ver(self, repo_ver_href):
 
     def import_and_invalidate(
             self, distribution_name, packages_to_add,
-            invalidate_expression, invalidate_downstream):
+            invalidate_expression, invalidate_downstream, package_cache={}, dry_run=False):
```

Comment:
Using a mutable type like a `dict` as a default value is dangerous since it will be using the same instance for multiple invocations. Use `None` instead.

Reply:
Even if the function is only reading from it?

Reply:
Yes.